operators.py

```diff
@@ -2996,6 +2996,8 @@ def main(self, context):
 
     def fini(self, context, succeede):
         if hasattr(self, "target"):
+            if hasattr(self.target, "inhibit"):
```

Comment:
this is odd... the target's Inhibit Flag seems to remain true as soon as we hit a breakpoint in class_defines.py invert_angle_setter() where the update continues to be inhibited (correctly).
But later, using the Context Menu, the Inhibit Flag is correctly False.
I'm not sure why this "works"